README.md

```diff
@@ -1 +1,21 @@
-# hw01
\ No newline at end of file
+# hw01
+
+### A Glimpse of My Life
+Welcome to my blurb! I'm Kelly Zhang and a rising second-year in the College. I'll be telling you a bit more about myself today.
+
+***Places of Residence*** I was born in St. Paul, MN and moved to Santa Fe, NM and later to the suburbs of Washington D.C., where I still reside today. 
```

Comment:
The use of bolded and italic words makes the structure of the doc really easy to read!